migration/migration.go

```diff
@@ -282,6 +283,9 @@ func GetMigrations() Migrations {
 	// Version 54
 	m = append(m, steps{ExecuteSQLFile("054-add-stackid-to-codebase.sql")})
 
+	// Version 55
+	m = append(m, steps{ExecuteSQLFile("055-space-templates.sql", "b5beea58-fa52-43b1-a21d-dd6a8d6e61bb")})
```

Comment:
To be honest with you, I want to get rid of it sooner than later and not keep it in around at all. This is the ID of the space template for all the spaces that currently exists in the system. I'm not sure we want to have spaces in the system at all after this PR because they have been defined with a space template which was given by the `migration.go` file. That implicit template isn't part of @qodfathr 's space templates. Let me think a little longer how to deal with this.